### DISCUSSION TITLE
Need Help with ASROCK  Z390 Phantom Gaming X

### DISCUSSION THREAD

**Discussion by Johnfrag** [terminatorul/NvStrapsReBar#65] [General]
Hi I need help modifying my bios, I have a ASROCK Z390 Phantom Gaming X 

I have an RTX 2080 and would like to activate the card's REZ.

I have currently installed the bios version 1.20N [Beta]

https://pg.asrock.com/mb/Intel/Z390%20Phantom%20Gaming%20X/index.asp#BIOS

Unfortunately I can't manage to modify the bios so that I can flash it via the bios. Can anyone help me with this?

**Comment by wyhzxy** [comment]
https://www.bilibili.com/video/BV1UJ4m1j75G/?spm_id_from=333.1391.0.0&vd_source=f899dcc9bb91dbeb029a6134dfe1b4b8
